sotodlib/preprocess/processes.py

```diff
@@ -1171,6 +1172,43 @@ def process(self, aman, proc_aman):
         tod_ops.t2pleakage.subtract_t2p(aman, proc_aman['t2p'],
                                         **self.process_cfgs)
 
+class RotateDemodQU(_Preprocess):
+    """Apply detectors' polarization angle rotation to the
+    demodulated Q and U timestreams to put them in the common
+    telescope frame.
+
+     Example config block::
+
+        - name : "rotate_demodQU"
+          process:
+            zero_gamma: True
+
+    .. autofunction:: sotodlib.coords.demod.rotate_demodQU
+    """
+    name = "rotate_demodQU"
+
+    def process(self, aman, proc_aman):
+        demod.rotate_demodQU(aman, **self.process_cfgs)
+
+class GetCommonMode(_Preprocess):
+    """Get common mode.
+
+     Example config block::
+
+        - name : "get_common_mode"
+          process:
+            signal: 'signal'
+            method: 'median'
+            wrap_name: 'signal_common_mode'
+
+    .. autofunction:: sotodlib.tod_ops.pca.get_common_mode
+    """
+    name = "get_common_mode"
+
+    def process(self, aman, proc_aman):
+        tod_ops.fft_ops.calc_psd(aman, **self.process_cfgs)
```

Comment:
This does not seem like the right function call...

Reply:
Ya the preprocess part of this PR needs work + testing. Kyohei left it up to me to fix this part so I'm holding that back. Will get to it soon apologies for the holdup!